htsohm/htsohm.py

```diff
@@ -139,7 +139,15 @@ def retest(m_orig, retests, tolerance):
         m_orig.retest_num += 1
 
         if m_orig.retest_num == retests:
-            m_orig.retest_passed = m.calculate_retest_result(tolerance)
+            while True:
+                try:
+                    m_orig.retest_passed = m.calculate_retest_result(tolerance)
+                except ZeroDivisionError as err:
```

Comment:
wait, why is a ZeroDivisionError being thrown? Does that make any sense? (and can we fix it at the source?)